internal/impl/sql/output_sql_insert.go

```diff
@@ -212,7 +219,7 @@ func (s *sqlInsertOutput) WriteBatch(ctx context.Context, batch service.MessageB
 		if tx, err = s.db.Begin(); err != nil {
 			return err
 		}
-		sqlStr, _, err := insertBuilder.Values().ToSql()
+		sqlStr, _, err := insertBuilder.ToSql()
```

Comment:
Any idea if this breaks Clickhouse? The Clickhouse tests do pass after this change, so I think it's fine, but not sure... What happens is that the current implementation produces a query like so: `INSERT INTO FOOTABLE (C1) VALUES ()` which Oracle doesn't like. After this change, the statement becomes `INSERT INTO FOOTABLE (foo) VALUES (:1)`.

Reply:
If the tests pass then I think we're fine, the original was likely me just trying stuff until it worked anyway.